api/v1alpha2/zz_generated.conversion.go

```diff
@@ -57,26 +57,6 @@ func RegisterConversions(s *runtime.Scheme) error {
 	}); err != nil {
 		return err
 	}
-	if err := s.AddGeneratedConversionFunc((*AWSClusterSpec)(nil), (*v1alpha3.AWSClusterSpec)(nil), func(a, b interface{}, scope conversion.Scope) error {
-		return Convert_v1alpha2_AWSClusterSpec_To_v1alpha3_AWSClusterSpec(a.(*AWSClusterSpec), b.(*v1alpha3.AWSClusterSpec), scope)
-	}); err != nil {
-		return err
-	}
-	if err := s.AddGeneratedConversionFunc((*v1alpha3.AWSClusterSpec)(nil), (*AWSClusterSpec)(nil), func(a, b interface{}, scope conversion.Scope) error {
-		return Convert_v1alpha3_AWSClusterSpec_To_v1alpha2_AWSClusterSpec(a.(*v1alpha3.AWSClusterSpec), b.(*AWSClusterSpec), scope)
-	}); err != nil {
-		return err
-	}
-	if err := s.AddGeneratedConversionFunc((*AWSClusterStatus)(nil), (*v1alpha3.AWSClusterStatus)(nil), func(a, b interface{}, scope conversion.Scope) error {
-		return Convert_v1alpha2_AWSClusterStatus_To_v1alpha3_AWSClusterStatus(a.(*AWSClusterStatus), b.(*v1alpha3.AWSClusterStatus), scope)
-	}); err != nil {
-		return err
-	}
-	if err := s.AddGeneratedConversionFunc((*v1alpha3.AWSClusterStatus)(nil), (*AWSClusterStatus)(nil), func(a, b interface{}, scope conversion.Scope) error {
-		return Convert_v1alpha3_AWSClusterStatus_To_v1alpha2_AWSClusterStatus(a.(*v1alpha3.AWSClusterStatus), b.(*AWSClusterStatus), scope)
-	}); err != nil {
-		return err
-	}
```

Comment:
Not quite sure why the new version of conversion-gen is removing these, at first it seemed like it was removing everything except the top level types, but that doesn't appear to be consistent as some non-top level types are still present.

Reply:
Ah, it looks like these were previously duplicated